scene/2d/remote_transform_2d.cpp

```diff
@@ -63,7 +63,21 @@ void RemoteTransform2D::_update_remote() {
 		return;
 
 	//todo make faster
-	n->set_global_transform(get_global_transform());
+	if (use_full_transform == true) {
```

Comment:
You don't need to check for `== true`, `if (use_full_transform)` works fine already.